qml/pages/CameraUI.qml

```diff
@@ -358,19 +359,27 @@ Page {
     }
 
     function setFocusMode(focus) {
-        if (camera.focus.focusMode !== focus) {
+        if (focus === Camera.FocusManual && !_manualModeSelected) {
```

Comment:
The extra condition "!_manualModeSelected" in this causes focus mode to change to Auto if you press manual mode a second time in selection menu.